lms/djangoapps/course_goals/models.py

```diff
@@ -0,0 +1,35 @@
+"""
+Course Goals Models
+"""
+from django.contrib.auth.models import User
+from django.db import models
+from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
+
+
+class CourseGoal(models.Model):
+    """
+    Represents a course goal set by a user on the course home page.
+
+    The goal_key represents the goal key that maps to a translated
+    string through using the CourseGoalOption class.
+    """
+    GOAL_KEY_CHOICES = (
```

Comment:
1. These can be lazily translated like the [following example](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/waffle_utils/models.py#L17).